nncf/torch/composite_compression.py

```diff
@@ -127,8 +127,3 @@ def prepare_for_export(self):
 
         for child_ctrl in self.child_ctrls:
             child_ctrl.prepare_for_export()
-
-    def apply_to(self, target_model: NNCFNetwork) -> NNCFNetwork:
-        for ctrl in self.child_ctrls:
-            target_model = ctrl.apply_to(target_model)
-        return target_model
```

Comment:
@vshampor Are there any concerns about these changes?

Reply:
Well, if these are just copies of what is defined in superclasses, then no.